tests/bluetooth/qualification/ICS_Zephyr_Bluetooth_Host.pts

```diff
@@ -2938,6 +3066,93 @@
         <row>4</row>
       </item>
     </profile>
+    <profile>
+      <name>LC3</name>
+      <item>

```

Comment:
Since we do not have our own LC3 codec, I don't think we should qualify it. Is is mandatory to add to the qualification?

Reply:
yeap, this is due to BAP
LC3 > BAP | If [CORE] (40/2) and [BAP] (93/1) are Supported then [LC3] (5/1) is Mandatory Go to Layer Selection.

but for true qualification one can refer to external QDID of their own codec, so this is just to make consistency check happy

Reply:
But we don't want to be running LC3 tests for Zephyr, as we do not provide that. Won't this enable a bunch of LC3 tests?

Reply:
no, LC3 is not tested with PTS (there are separate tools for that)